components/base_page.js

```diff
@@ -11,10 +12,23 @@ import css from '../styles/common.module.scss';
 import utilsCss from '../styles/utils.module.scss';
 
 export default function Page({ children }) {
+    const router = useRouter();
+    const path = router.pathname;
+    console.log(path);
```

Comment:
please remove `console.log` statements before committing/PRs

Reply:
I removed the console.log.